src/util.ts

```diff
@@ -35,6 +36,14 @@ function checkIsMinGW(): boolean {
   }
 }
 
+/**
+ * Wrapper around globby that defaults the option to use gitignore
+ */
+export async function resolveGlobs(globs: string[]) {
+  // TODO(stramel): globby options typing is incorrect
+  return await globby(globs, {gitignore: true} as any);
```

Comment:
this gitignore feels a bit unexpected, I could easily see users confused why files weren't being matched. Can we remove it?

Reply:
Yeah, we can and at that point, probably no need for the wrapper function?

Reply:
I could go either way on adding this sort of extra functionality to the analyzer. Agreed that it's useful, but it's also extra complexity/dependencies/maintenance/potential for bugs, for a feature that most people won't need.

Either way, this is really good, lets not block it here.